src/public/main.module.js

```diff
@@ -16,7 +16,7 @@ import 'file-loader?name=fonts/[name].[ext]!./fonts/bb-icons.svg';
 import './images/bb-logo.svg';
 
 // styles
-import './main.scss';
+//import './main.scss';
```

Comment:
Does this commented line need to be here?

Reply:
Absolutely not!